License identified in include/linux/tcp.h:
/* SPDX-License-Identifier: GPL-2.0-or-later */

--- a/include/linux/tcp.h.orig	2023-11-08 17:23:07.253366398 -0800
+++ b/include/linux/tcp.h	2023-11-08 17:25:06.549368830 -0800
@@ -277,7 +277,8 @@ struct tcp_sock {
 	u8	dup_ack_counter:2,
 		tlp_retrans:1,	/* TLP is a retransmission */
 		tcp_usec_ts:1, /* TSval values in usec */
-		unused:4;
+		fast_ack_mode:2, /* which fast ack mode ? */
+		unused:2;
 	u32	chrono_start;	/* Start time in jiffies of a TCP chrono */
 	u32	chrono_stat[3];	/* Time in jiffies for chrono_stat stats */
 	u8	chrono_type:2,	/* current chronograph type */
